src/api/app/controllers/webui/obs_factory/staging_projects_controller.rb

```diff
@@ -38,6 +38,8 @@ def show
           @staging_project = ::ObsFactory::StagingProjectPresenter.new(@staging_project)
           # For the breadcrumbs
           @project = @distribution.project
+          build_id = Backend::Api::Published.build_id(@staging_project.name, 'images')
```

Comment:
Not sure `images` should be hardcoded...

Reply:
I added a FIXME. For the staging dashboard only images repository makes sense for now but we need to discuss that with the people who will actually use it in the future.

Reply:
Okidoke